src/model_theory/basic.lean

```diff
@@ -56,34 +61,51 @@ protected def empty : language := ⟨λ _, pempty, λ _, pempty⟩
 
 instance : inhabited language := ⟨language.empty⟩
 
-/-- The type of constants in a given language. -/
-@[nolint has_inhabited_instance] def const (L : language) := L.functions 0
+/-- The sum of two languages consists of the disjoint union of their symbols. -/
+protected def sum (L : language.{u v}) (L' : language.{u' v'}) : language.{(max u u') (max v v')} :=
+⟨λn, L.functions n ⊕ L'.functions n, λ n, L.relations n ⊕ L'.relations n⟩
 
 variable (L : language.{u v})
 
+/-- The type of constants in a given language. -/
+@[nolint has_inhabited_instance] def const := L.functions 0
```

Comment:
```suggestion
@[nolint has_inhabited_instance] def constants := L.functions 0
```

Reply:
This doesn't work, `constants` is a Lean command. Would pluralizing to `consts`, or some other compromise, be better though?

Reply:
I see. `consts` is already better, yeah.

Reply:
You can still use the name `constants`, if you write `protected def «constants» := ...` or `protected def _root_.language.constants := ...`.
In both cases you can then use it with `L.constants` (the `def` should be protected in this case, since `constants` by itself will not be accepted by Lean.